doc/auto_techsupport_and_coredump_mgmt.md

```diff
@@ -226,27 +298,43 @@ module sonic-auto_techsupport {
 #### AUTO_TECHSUPPORT_DUMP_INFO Table

```

Comment:
>Table [](http://example.com/codeflow?start=32&length=5)

Is there an entry number limit in this table?

Reply:
@qiluo-msft There is a rate limiting mechanism already available in auto techsupport infrastructure that limits the number of dumps that can be generated.